packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/ios/RunnerTests/GoogleMapsGroundOverlayControllerTests.m

```diff
@@ -191,4 +185,100 @@ - (void)testUpdatingGroundOverlayWithBounds {
   XCTAssertEqual(convertedPlatformGroundOverlay.zIndex, platformGroundOverlay.zIndex);
 }
 
+- (void)testUpdateGroundOverlaySetsVisibilityLast {
+  PropertyOrderValidatingGroundOverlay *groundOverlay =
+      [[PropertyOrderValidatingGroundOverlay alloc] init];
+  [FGMGroundOverlayController
+            updateGroundOverlay:groundOverlay
+      fromPlatformGroundOverlay:
+          [FGMPlatformGroundOverlay
+              makeWithGroundOverlayId:@"groundOverlay"
+                                image:[FGMPlatformBitmap
+                                          makeWithBitmap:[FGMPlatformBitmapDefaultMarker
+                                                             makeWithHue:@0]]
+                             position:[FGMPlatformLatLng makeWithLatitude:0 longitude:0]
+                               bounds:[FGMPlatformLatLngBounds
+                                          makeWithNortheast:[FGMPlatformLatLng
+                                                                makeWithLatitude:54.4816
+                                                                       longitude:5.1791]
+                                                  southwest:[FGMPlatformLatLng
+                                                                makeWithLatitude:52.4816
+                                                                       longitude:3.1791]]
+                               anchor:[FGMPlatformPoint makeWithX:0.5 y:0.5]
+                         transparency:0.5
+                              bearing:65.0
+                               zIndex:2.0
+                              visible:YES
+                            clickable:YES
+                            zoomLevel:nil]
+                    withMapView:[GoogleMapsGroundOverlayControllerTests mapView]
+                      registrar:nil
+                    screenScale:1.0
+                    usingBounds:YES];
+  XCTAssertTrue(groundOverlay.hasSetMap);
+}
+
+/// Returns a simple map view to add map objects to.
++ (GMSMapView *)mapView {
+  GMSMapViewOptions *mapViewOptions = [[GMSMapViewOptions alloc] init];
+  mapViewOptions.frame = CGRectMake(0, 0, 100, 100);
+  mapViewOptions.camera = [[GMSCameraPosition alloc] initWithLatitude:0 longitude:0 zoom:0];
+  return [[PartiallyMockedMapView alloc] initWithOptions:mapViewOptions];
+}
```

Comment:
We could extract this as a convenience constructor, but it was simple enough that I just copied it from the tests that already did it into each other test file.